src/core/adapters/github.ts

```diff
@@ -38,16 +38,21 @@ async function attempt(
   if ($has(select.issuesPage, doc)) {
     const issues = $all(select.issuesPage, doc);
 
-    const tickets = issues.map((issue) => {
+    const tickets = issues.reduce((acc, issue) => {
```

Comment:
Turned this into a `reduce`, so that we can add the check for `id` and `title` to potentially "skip" an entry. Makes the compiler happy, but is unlikely to ever happen in the real world.